apps/dialtone-documentation/docs/components/stack.md

```diff
@@ -156,6 +156,45 @@ vueCode='
 '
 showHtmlWarning />
 
+Set gap to 600 at extra large screen size, 500 for large screen size, 400 for medium screen size, and 300 for small screen size
+
+<code-well-header>
+  <dt-stack :gap="{ xl: '600', lg: '500', md: '400', sm: '300' }">
+    <div class="d-bgc-magenta-100">
+      Stack item 1
+    </div>
+    <div class="d-bgc-magenta-100">
+      Stack item 2
+    </div>
+    <div class="d-bgc-magenta-100">
+      Stack item 3
+    </div>
+  </dt-stack>
+</code-well-header>
+
+<code-example-tabs
+htmlCode='
+<div class="d-stack d-stack--sm-gap-300 d-stack--md-gap-400 d-stack--lg-gap-500 d-stack--xl-gap-600">
+  <div class="d-bgc-magenta-100">Stack item 1</div>
+  <div class="d-bgc-magenta-100">Stack item 2</div>
+  <div class="d-bgc-magenta-100">Stack item 3</div>
+</div>
+'
+vueCode='
+<dt-stack :gap="{ xl: `600`, lg: `500`, md: `400`, sm: `300` }">
```

Comment:
Much like `direction` has...
```
:direction="{ `default`: `row`, `sm`: `column`, `lg`: `column-reverse` }"
```
...I was expecting there to be a `default` for `gap` as well, like so...
```
:gap="{ `default`: `600`, `lg`: `300` }"
```
That reads as:
"`600` gap by default. If viewport is `lg` or smaller, the gap is `300` (including for `md` and `sm`)."

Another example:
```
:gap="{ `default`: `600`, `lg`: `300`, `sm`: `100`  }"
```
In that one, the `lg` gap of `300` applies to `lg` **and** `md`, and then the `sm` value overrides that.

So yeah, our media queries are backward by going downward in an **overriding** fashion, rather than **upward**. Changing that for all media queries is a massive breaking change, and is part of the debt we've inherited from Dialtone's initial creation.

Reply:
The expected then is that each breakpoint represents the breakpoint or smaller? What is the expected then for the largest breakpoint, xl?

Reply:
> The expected then is that each breakpoint represents the breakpoint or smaller?

Yeah, that's the "backward" part of RWD we've inherited. I hate it.

> What is the expected then for the largest breakpoint, xl?

That would be `default`, just like the Stack's `direction` prop has. It's not a mq breakpoint, per se, it's just the default it starts with until a narrower breakpoint is declared to do something different. Screencast of `direction` prop, which is what I expect `gap` to follow:

https://github.com/user-attachments/assets/0ed8fc91-7d98-4dd9-ba81-a12344aad828

Reply:
Oh, I understand now. The default direction is column, but the default gap is 0, that's why you won't get any gap is you don't explicitly define a default. 

<img width="916" alt="image" src="https://github.com/user-attachments/assets/7594d924-3b43-40e5-87af-0c6b6bc0d7a0">